docs/guides/qunova-chemistry.ipynb

```diff
@@ -115,7 +115,8 @@
     "from qiskit_ibm_catalog import QiskitFunctionsCatalog\n",
     "\n",
     "catalog = QiskitFunctionsCatalog()\n",
-    "function = catalog.load(\"qunova/hivqe-chemistry\")"
+    "function = catalog.load(\"qunova/hivqe-chemistry\")\n",
+    "instance = \"\"  # replace this with your authenticated instance string, for example, \"ibm-q/open/main\" for the open plan"
```

Comment:
```suggestion
    "instance = \"\"  # replace this with your authenticated instance string"
```
(removing h/g/p language since it only applies to IBM Quantum Platform Classic)